app/assets/stylesheets/minimal.scss

```diff
@@ -37,6 +37,7 @@
 @import 'typography';
 @import 'buttons';
 @import 'widgets';
+@import 'curated-clusters-modal';
```

Comment:
this is where you are adding all the style designs for the curated clusters?

Reply:
Yep! If you are not familiar with what SCSS is it is a way of adding some extra functionality onto normal css. We can use functions, mixins, imports to augment the structure of our css.